tools/linkage_disequilibrium/README.md

```diff
@@ -0,0 +1,128 @@
+The script calculats LD between variants using genotypes from a selected population. We only support LD calculation for variants for which we have genotypes from at least 40 samples in the selected population. At the moment we only have sufficient amounts of genotype data from the 1000 Genomes project for human.
+
+There is a [web tool](http://www.ensembl.org/Multi/Tools/LD) version with an interface for easy parameter selection. 
+
+##### Table of contents
+* [Usage](#usage)
+* [Show all populations](#show_all_populations)
+* [LD calculations](#ld_calculations)
+* [Input file formats](#input_file_formats)
+* [Output file formats](#output_file_formats)
+* [Examples](#examples)
+---
+<a name="usage"></a>
+### Usage
+The script wraps around the LD calculation code in the Ensembl Variation API. Before running the ld_tool script follow the [installation instructions](https://github.com/Ensembl/ensembl-variation/blob/master/C_code/README.txt).
+
+#### Running the script
+```bash
+./ld_tool -i input_file --calculation [center|pairwise|region] 
+```
+* `-i|--input_file [input_file]` : [input_file](#input_file_formats)
+* `-c|--calculation [center|pairwise|region]` : [calculation](#ld_calculation)
+
+#### Script options
+* `--species [species]` : species to use (default: human)
+* `--assembly [assembly]` : assembly to use (If human is used default assembly is grch38)
+* `--output_file [output_file]` : If not provided output_file name is set to [input_file].out or if input_file is not provided to [calculation]_year_month_day_hour_minutes.out
+* `--warnings_file [warnings_file]` : If not provided warnings_file name is set to [input_file].warnings or if input_file is not provided to [calculation]_year_month_day_hour_minutes.warnings
+* `--population [population_name]` : List of population(s) for LD calculation
+* `--region [region]` : List of region(s) as input for region calculation. A region is defined as chromosome:start-end
+* `--variant [variant]` : List of variant(s) as input for either center or pairwise calculations
+* `--r2 [r2]` : Only include variants to the result whose r2 value is greater than or equal to the given value. r2 needs to be in the range of 0.0 and 1.0.
+* `--d_prime [d_prime]` : Only include variants to the result whose d_prime value is greater than or equal to the given value. d_prime needs to be in the range of 0.0 and 1.0.
+* `--add_variant_attribs` : Add variant attributes (evdidence values and consequence type) to the output
+
+<a name="show_all_populations"></a>
+### Show all populations
+Show all populations that can be used for the given species. 
+```bash
+./ld_tool --show_all_populations --species species
+```
+<a name="ld_calculations"></a>
+### LD calculations
+The script supports three different types of caclulations.
+#### Center
+Compute all pairwise LD values for a given variant and all variants that are not further away from the given variant than the selected window size which centers around the input variant.
```

Comment:
Is this a little unclear? Not sure if this is better-
Compute all pairwise LD values for a given variant with all variants within a window of the specified size, centered on the input variant.